src/autocomplete.js

```diff
@@ -86,6 +86,19 @@ class Autocomplete {
         this.stickySelectionTimer = lang.delayedCall(function() {
             this.stickySelection = true;
         }.bind(this), this.stickySelectionDelay);
+
+        this.$firstOpenTimer = lang.delayedCall(function() {
+            var initialPosition = this.completionProvider && this.completionProvider.initialPosition;
+            if (this.autoShown || (this.popup && this.popup.isOpen) || !initialPosition) return;
+
+            var completionsForEmpty = [{
+                caption: "Loading...",
```

Comment:
We should allow this string to be translated through the `nls` function we added.

Reply:
done